README.md

```diff
@@ -23,13 +23,25 @@ Developed for use with [Alchemy](https://github.com/dawnlabs/alchemy)
 ### Convert
 
   To and from: `jpg`, `png`, `gif`, `bmp`, `tiff`
-  
-### Merge 
+
+### Merge
 
   To: `pdf`
-  
+
   From: `jpg`, `png`, `gif`
 
+## Development
+
+Photosorcery has been created to be used with an electron application called [Alchemy](https://github.com/dawnlabs/alchemy) which is built to run on Linux, Windows as well as MacOS. This requires Photosorcery having to be built for each of those architectures.
+
+Some of Photosorcery's dependencies require cross compilation which we achieve with the help of Docker and https://github.com/karalabe/xgo.
+
+When developing:
+
+* Clone this repository somewhere in your `$GOPATH`
+* Install [Docker](https://www.docker.com/community-edition#/download)
```

Comment:
Can we put _Install Docker_ first? Contextually, the other two seem linked to me (clone the repo, then run make) so maybe putting them next to each other would be easier to follow.